anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc

```diff
@@ -27,7 +27,10 @@ When a consumer takes a message from the queue and stores it in the local buffer
 +
 The smaller buffer restricts app-level throughput and avoids blocking competing consumers.
 
-When using the `prefetch` mode, the `acknowledgementTimeout` timer begins from the moment the message is dispatched to the flow, not when it is taken from the queue. The time the message remains in the local buffer doesn't count as part of the timeout.
+When using the `prefetch` mode, the `acknowledgementTimeout` timer begins from the moment the message is taken from the queue. The time the message remains in the local buffer doesn't count as part of the timeout.
```

Comment:
Same here, it actually does, the message in the local buffer counts as part of the timeout except when we reach the previously mentioned condition